src/lib/addons/addon.ts

```diff
@@ -75,15 +71,13 @@ export default abstract class Addon {
             this.logger.warn(
                 `Error querying ${url} with method ${
                     method || 'GET'
-                } status code ${e.code}`,
+                } status code ${(e as any).code}`,
```

Comment:
Pretty sure this cast isn't necessary

```suggestion
                } status code ${e.code}`,
```